src/main/java/duke/command/ExitCommand.java

```diff
@@ -0,0 +1,32 @@
+package duke.command;
+
+import duke.storage.Storage;
+import duke.tasklist.TaskList;
+import duke.ui.Ui;
+
+/**
+ * Represents a user command to exit the Duke program.
+ */
+public class ExitCommand extends Command {
+    private String userInput;
+
+    public ExitCommand(String userInput) {
+        this.userInput = userInput;
+    }
+
+    public boolean isExit() {
+        return true;
+    }
+
+    /**
+     * Exits the Duke program.
```

Comment:
Maybe change the description to "prints goodbye text"? Currently the code block doesn't really exit the program.